.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -39,4 +39,4 @@ body:
     id: extra-context
     attributes:
       label: Extra information or context
-      description: Add any other context about the problem here.
+      description: IF YOU ARE REPORTING A REGRESSION: Use `ujust rebase-helper` to find the version that used to work. Report the working version and the broken version so that other users can rebase and confirm the issue!
```

```suggestion
      description: If you are reporting a regression: Use `ujust rebase-helper` to find the version that used to work. Report the working version and the broken version so that other users can rebase and confirm the issue!
```
This sounds oddly aggressive, should we do that?